ONSdigital/eq-author-app
# Ear 646 expression validation AJV setup

Context?

Adds validation checks on expressions (routing, skipConditions) and passes errors to the front end

Currently works with empty rules for both skip and routing

How to review?

- Create a questionnaire
- Add both skip and routing rules
- Go to the network tabs and find the graphql responses for both skip and routing or graphql reader
- Drill down into the expression for each and check if there is a validationErrorInfo present
- Find the graphql response for the page as a whole. In the ValidationErrorInfo, all errors should be present
- The design tab should also show errors are present
